composer.json

```diff
@@ -13,72 +13,78 @@
     ],
     "homepage": "https://laminas.dev",
     "support": {
-        "docs": "https://docs.laminas.dev/laminas-servicemanager/",
         "issues": "https://github.com/laminas/laminas-servicemanager/issues",
-        "source": "https://github.com/laminas/laminas-servicemanager",
-        "rss": "https://github.com/laminas/laminas-servicemanager/releases.atom",
+        "forum": "https://discourse.laminas.dev",
         "chat": "https://laminas.dev/chat",
-        "forum": "https://discourse.laminas.dev"
-    },
-    "config": {
-        "platform": {
-            "php": "8.0.99"
-        },
-        "sort-packages": true,
-        "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true,
-            "composer/package-versions-deprecated": true,
-            "laminas/laminas-dependency-plugin": true
-        }
+        "source": "https://github.com/laminas/laminas-servicemanager",
+        "docs": "https://docs.laminas.dev/laminas-servicemanager/",
+        "rss": "https://github.com/laminas/laminas-servicemanager/releases.atom"
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
-        "laminas/laminas-stdlib": "^3.2.1",
+        "php": "~8.1.0 || ~8.2.0",
+        "laminas/laminas-stdlib": "^3.17",
         "psr/container": "^1.0"
     },
     "require-dev": {
         "composer/package-versions-deprecated": "^1.11.99.5",
+        "friendsofphp/proxy-manager-lts": "^1.0.14",
+        "laminas/laminas-code": "^4.10.0",
         "laminas/laminas-coding-standard": "~2.5.0",
         "laminas/laminas-container-config-test": "^0.8",
         "laminas/laminas-dependency-plugin": "^2.2",
-        "mikey179/vfsstream": "^1.6.11@alpha",
-        "ocramius/proxy-manager": "^2.14.1",
-        "phpbench/phpbench": "^1.2.7",
-        "phpunit/phpunit": "^9.5.26",
-        "psalm/plugin-phpunit": "^0.18.0",
-        "vimeo/psalm": "^5.0.0"
+        "mikey179/vfsstream": "^1.6.11",
+        "phpbench/phpbench": "^1.2.9",
+        "phpunit/phpunit": "^9.6.5",
+        "psalm/plugin-phpunit": "^0.18.4",
+        "vimeo/psalm": "^5.8.0"
     },
-    "provide": {
-        "psr/container-implementation": "^1.0"
+    "replace": {
+        "container-interop/container-interop": "^1.2.0"
     },
     "conflict": {
         "ext-psr": "*",
-        "laminas/laminas-code": "<3.3.1",
+        "laminas/laminas-code": "<4.10.0",
         "zendframework/zend-code": "<3.3.1",
         "zendframework/zend-servicemanager": "*"
     },
+    "provide": {
+        "psr/container-implementation": "^1.0"
+    },
     "suggest": {
-        "ocramius/proxy-manager": "ProxyManager ^2.1.1 to handle lazy initialization of services"
+        "friendsofphp/proxy-manager-lts": "ProxyManager ^2.1.1 to handle lazy initialization of services"
```

Comment:
The documentation must also be updated: https://docs.laminas.dev/laminas-servicemanager/lazy-services/#setup

Reply:
Ops, thank you :+1: